angular.json

```diff
@@ -130,24 +134,28 @@
                 "input": "src/highlightjs/material-light.scss"
               },
               {
+                "lazy": true,
                 "inject": false,
-                "input": "src/assets/pink-bluegrey.css",
-                "bundleName": "pink-bluegrey"
+                "input": "src/styles/custom-themes/pink-bluegrey.scss",
+                "bundleName": "assets/pink-bluegrey"
               },
               {
+                "lazy": true,
                 "inject": false,
-                "input": "src/assets/deeppurple-amber.css",
-                "bundleName": "deeppurple-amber"
+                "input": "src/styles/custom-themes/deeppurple-amber.scss",
+                "bundleName": "assets/deeppurple-amber"
               },
               {
+                "lazy": true,
                 "inject": false,
-                "input": "src/assets/indigo-pink.css",
-                "bundleName": "indigo-pink"
+                "input": "src/styles/custom-themes/indigo-pink.scss",
+                "bundleName": "assets/indigo-pink"
               },
               {
+                "lazy": true,
```

Comment:
`lazy` is the old deprecated name for `inject`, we don't need both.

Can you please try rebasing your branch on `master` as well?